operator/src/test/java/oracle/kubernetes/weblogic/domain/model/ServerStatusTest.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+public class ServerStatusTest {
+
+  static ServerStatus cluster1Server1 = new ServerStatus().withClusterName("cluster-1").withServerName("server1");
+  static ServerStatus cluster1Server2 = new ServerStatus().withClusterName("cluster-1").withServerName("server2");
+  static ServerStatus cluster1Server10 = new ServerStatus().withClusterName("cluster-1").withServerName("server10");
+
+  static ServerStatus cluster2Server1 = new ServerStatus().withClusterName("cluster-2").withServerName("server1");
+  static ServerStatus cluster10Server1 = new ServerStatus().withClusterName("cluster-10").withServerName("server1");
+
+  static ServerStatus standAloneServer1 = new ServerStatus().withServerName("server1");
+  static ServerStatus standAloneServer2 = new ServerStatus().withServerName("server2");
+  static ServerStatus standAloneServer10 = new ServerStatus().withServerName("server10");
+
+  static ServerStatus adminServer = new ServerStatus().withServerName("admin-server").withIsAdminServer(true);
+  static ServerStatus cluster1ServerA = new ServerStatus().withClusterName("cluster-1").withServerName("a");
+  static ServerStatus standAloneServerA = new ServerStatus().withServerName("a");
+
+  static ServerStatus nullClusterNullServer = new ServerStatus();
+
+  @Test
+  public void verify_Equal_compareTo() {
+    assertThat(nullClusterNullServer.compareTo(nullClusterNullServer), equalTo(0));
```

Comment:
these asserts should probably have messages on them ?

Reply:
https://jira.oraclecorp.com/jira/browse/OWLS-80945

Reply:
this method tests that compareTo should return 0 when comparing to the same object. The assertThat statement seems to be self explanatory.